src/project.cpp

```diff
@@ -2097,6 +2099,24 @@ bool Project::readTilesetProperties() {
         logWarn(QString("Value for tileset property 'NUM_PALS_TOTAL' not found. Using default (%1) instead.")
                 .arg(Project::num_pals_total));
     }
+    it = defines.find("MAX_MAP_DATA_SIZE");
```

Comment:
This should probably be its own function.  It doesn't make sense to put it in `readTilesetProperties()`